### PR TITLE
XS✔ ◾ added devx in microservices rule

### PR DESCRIPTION
**Pull request by lukecookssw**
>
> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

PTSD from Microservices

> 2. What was changed?

New rule about priotizing DevX in Microservices

> 3. Did you do pair or mob programming (list names)?

### CODE REVIEW

File: rules/prioritize-devx-in-microservices/rule.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+* Comprehensive and up-to-date documentation is essential. Each microservice should have clear and concise documentation that covers:
+  * API endpoints and their usage
+  * Configuration settings
+  * Deployment instructions
+  * Common troubleshooting steps
+
+### Keep it accessible
+
+* Documentation should be easily accessible to all team members. The recommended way is to have a great `readme` file at the top level of your repo. See our [awesome documentation rules](awesome-documentation) for great tips on what to include!
```
Comment (danielmackay) [inline rules/prioritize-devx-in-microservices/rule.md:32]: Where should this be if your services are split over many repos?

**Review by danielmackay — CHANGES_REQUESTED** [review]
Few comments

File: rules/prioritize-devx-in-microservices/rule.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+type: rule
+archivedreason:
+title: Do you prioritize DevX in Microservice Architecture?
+guid: 89c22fed-c9ce-432a-ae2a-419c95c95f31
+uri: prioritize-devx-in-microservices
+created: 2024-10-25T21:54:17.0000000Z
+authors:
+  - title: Luke Cook
+    url: https://ssw.com.au/people/luke-cook
+related: []
```
[the remark this answers is not in the document]
Comment (lukecookssw) [reply rules/prioritize-devx-in-microservices/rule.md:12]: Done

File: rules/prioritize-devx-in-microservices/rule.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
+---
+seoDescription: Prioritize DevX when working with Microservice Architecture
```
[the remark this answers is not in the document]
Comment (lukecookssw) [reply rules/prioritize-devx-in-microservices/rule.md:2]: Done

File: rules/prioritize-devx-in-microservices/rule.md (hunk 1 above)
Comment (lukecookssw) [reply rules/prioritize-devx-in-microservices/rule.md:35]: Done

**Review by danielmackay — APPROVED** [review]
LGTM

**Pull request merged**